src/Type/Php/ArrayChunkFunctionReturnTypeExtension.php

```diff
@@ -46,7 +48,8 @@ public function getTypeFromFunctionCall(FunctionReflection $functionReflection,
 			$preserveKeys = false;
 		}
 
-		if ($lengthType instanceof ConstantIntegerType && $lengthType->getValue() < 1) {
```

Comment:
primary motivation was getting rid of the `instanceof ConstantIntegerType` parts in this extension